### DISCUSSION TITLE
no mixer device with hifiberry digi +

### DISCUSSION THREAD

**Discussion by rafverhoogen** [rern/rAudio#1767] [Bug Reports]
when activating the HifiBerry digi + hat i have no mixer device available anymore ?
i have seen similar remarks however updating did not solve the problem

**Comment by klassikmann** [comment]
@rafverhooge

I'm using a "Hifiberry Digi+ Pro" in the following configuration:
<img src="https://github.com/user-attachments/assets/11db236c-edc8-440c-9e70-5d6ec88e0d34" width=450>

That means, not using a mixer device, neither a hardware mixer nor a software mixer. The digital output of the "Digi+ Pro" is input to my active speakers, volume control is performed by a separate device. From a sound quality point of view this is the best configuration. IIRC a hardmare mixer isn't provided by the "Digi+", remains a software mixer provided by the MPD software, again from a sound quality point of view the worst configuration. 

I just unchecked the "No volume" check-box to see what's happening. But the software mixer seemed not to be recognized.

**Reply by rafverhoogen** [reply]
ok i understand 
however i am not getting any sound at all ? 
also arer there any special requirements for a usb drive to be mounted automatically at boot ?  in rune audio the same disk mounts without problems

**Reply by klassikmann** [reply]
@rafverhoogen 
Did you ever - using older releases of rAudio - get sound with the Digi+?

Hifiberry offers four product groups (for details look at their web-site):
- Analog
These are the various DACs providing digital-to-analog conversion to be connected to an external amplifier
- Amplifiers
Combining DAC and amplification to be connected to external analog loudspeakers.
- Digital
(as your Digi+) providing S/PDIF-Toslink audio interfaces to be connected to external devices with digital Koax- and/or Optical-Inputs like digital Amplifiers of digital speakers providing the digital-to-analog conversion.
- DSP
Enhancing analog products (DAC) with digital signal processing.

So, the main question is your overall audio environment. This defines the product to choose.

**Comment by rern** [comment]
@klassikmann,

The Mixer Device should be hidden if there's no mixers.
Please post output of: `amixer -c 0 scontents`

**Reply by klassikmann** [reply]
@rern 
Using my "Digi+ Pro":
```
#  amixer -c 0 scontents
Simple mixer control 'Tx Source',0
  Capabilities: enum
  Items: 'S/PDIF RX' 'AIF'
  Item0: 'AIF'
```
@rafverhoogen 
Please post output on your system.

**Reply by rafverhoogen** [reply]
@klassikmann 
shame is mine.
When i removed my cd transport from the system for some reason i changed the output of the PI to another input on the Devialet. Result it could not produce sound since it had no input :-).  i am very sorry for this confusion.
Also since i let the library build completly the usb seems to mount ok 

the good news is that all is working the bad news is that i feel like an idiot :-)

**Reply by klassikmann** [reply]
@rafverhoogen 
The main thing is that the problem could be solved quickly.
